Translations/pt_BR.po

```diff
@@ -7332,14 +7282,12 @@ msgstr ""
 "mesmo quando o modo amistoso está habilitado."
 
 #: Source/options.cpp:1056
-#, fuzzy
-#| msgid "No pause in multiplayer"
 msgid "Full quests in Multiplayer"
-msgstr "Não é permitido pausar no modo multijogador"
+msgstr "Todas as quests no modo Multijogador"
 
 #: Source/options.cpp:1056
 msgid "Enables the full/uncut singleplayer version of quests."
-msgstr ""
+msgstr "Habilita todas as quests de um jogador no modo multiplayer."
```

Comment:
Not a fan of this one. Alternative:
"Habilita a versão completa/sem cortes das missões de um jogador."